src/frontend/web_application/src/components/MessageList/components/Message/presenter.jsx

```diff
@@ -1,72 +1,85 @@
 import React, { PropTypes, Component } from 'react';
-import { v1 as uuidV1 } from 'uuid';
 import classnames from 'classnames';
+import { v1 as uuidV1 } from 'uuid';
+import VisibilitySensor from 'react-visibility-sensor'; // https://github.com/joshwnj/react-visibility-sensor
 import Moment from 'react-moment';
 import ContactAvatarLetter from '../../../ContactAvatarLetter';
-import Dropdown, { withDropdownControl } from '../../../../components/Dropdown';
 import Button from '../../../Button';
 import Icon from '../../../Icon';
 import TextBlock from '../../../TextBlock';
 import MultidimensionalPi from '../../../MultidimensionalPi';
-
 import MessageActionsContainer from '../MessageActionsContainer';
 
 import './style.scss';
 
 const FOLD_HEIGHT = 80; // = .m-message__content--fold height
 
-const DropdownControl = withDropdownControl(Button);
+function generateStateFromProps(props) {
+  const { message } = props;
+
+  return {
+    isRead: !message.is_unread,
```

Comment:
there is no reason to have `isRead` here since it is only useful for the checkbox that is in an other component